lesson-15/children-with-adults.js

```diff
@@ -0,0 +1,21 @@
+export function childrenWithoutAdults () {
+  if (counterAdult.value === '0') {
+    counterChildren.value = '0';
+    filterForm.style.height = '158px';
+    filterForm.style.width = '294px';
+    childAgeQuastion.classList.add('hide');
+    
+    btnCounterChildrenPlus.classList.add('cursor-not-allowed');
+    btnCounterChildrenPlus.classList.add('counter-plus-minus-gray');
+    btnCounterChildrenPlus.classList.remove('counter-plus-minus-blue');
+    btnCounterChildrenMinus.classList.remove('counter-plus-minus-blue');
+    btnCounterChildrenMinus.classList.add('counter-plus-minus-gray');
+    btnCounterChildrenPlus.setAttribute("disabled", "disabled");
+    counterChildren.setAttribute("disabled", "disabled"); 
+  } else {
+    btnCounterChildrenPlus.classList.remove('cursor-not-allowed');
+    btnCounterChildrenPlus.classList.add('counter-plus-minus-blue');
+    btnCounterChildrenPlus.removeAttribute("disabled");
+    counterChildren.removeAttribute("disabled");
+  };
+};
```

Comment:
The module should not depend on other modules. Here you just took some of the code and put it in a function, but it is not clear function. The function should not depend on variables outside of it. Do not use variables in functions that are not or are not passed as arguments
![image](https://user-images.githubusercontent.com/20189313/143313726-20626361-bd04-4633-aab0-f13bf1d41407.png)